.github/workflows/node-ci.yml

```diff
@@ -79,6 +79,13 @@ jobs:
         id: changed
       - run: echo '${{ steps.changed.outputs.files }}'
 
+      # Test json-args with a failing command
+      - uses: ./actions/json-args
+        continue-on-error: true
+        with:
+          list: '["hi","folks with space"]'
```

Comment:
NIT: Could you perhaps make it clear what these are? As in, is the `hi` the command that might run or just an arg? Is the `folks with space` also an arg?

Perhaps something like this may be clearer for future maintenance?

```
list: '["arg1_hi", "arg2_now with spaces"]'
```